weave-js/src/components/PagePanelComponents/Home/Browse3/pages/PlaygroundPage/PlaygroundSettings/PlaygroundSettings.tsx

```diff
@@ -59,7 +59,8 @@ export const PlaygroundSettings: React.FC<PlaygroundSettingsProps> = ({
     <div className="relative flex h-full w-[320px] shrink-0 flex-col border-l border-moon-250 pb-4">
       <div className="flex items-center justify-between gap-8 border-b border-moon-250 px-16 py-8">
         {/* Header */}
-        <div className="flex items-center gap-8">
+        {/* [&>span]: adds line-clamp to the child span from the tooltip*/}
+        <div className="line-clamp-1 flex items-center gap-8 [&>span]:line-clamp-1">
```

Comment:
I think there might be a slightly simpler approach here by specifying `noTriggerWrap` on the `<Tooltip>` and `w-full` on this flex container.